lua/modules/utils/init.lua

```diff
@@ -249,4 +249,38 @@ function M.tobool(value)
 	end
 end
 
+---@param plugin_name string @Plugin name
+---@param opts table @Default options for plugin
+---@param isVimPlugin? boolean @If plugin is made by vimscript
+---@param extraSetupFunc? function @When the plugin name that calls setup is different from `modules/configs/*.lua`
+function M.load_plugin(plugin_name, opts, isVimPlugin, extraSetupFunc)
```

Comment:
```suggestion
---@param plugin_name string @Name of the plugin
---@param opts table @Default options for plugin
---@param vim_plugin? boolean @If this plugin is a Vimscript one
---@param setup_callback? function @Provide this if the plugin's setup function is not the usual one
function M.load_plugin(plugin_name, opts, vim_plugin, setup_callback)
```
Better to be consistent in naming here.